jazzhands/__main__.py

```diff
@@ -196,18 +207,24 @@ def main(argv):
 
         if index_files['js'] and js_dir:
             jsx_registry_path = os.path.join(os.path.dirname(index_files['js']), 'jsx_registry.js')
-            subprocess.call(['python', 'manage.py', 'compilejsx'], stdout=open(jsx_registry_path, 'w'))
-            process_jsx(jsx_registry_path)
-            
+            try:
+                import django_jsx
+                subprocess.call(['python', 'manage.py', 'compilejsx'], stdout=open(jsx_registry_path, 'w'))
+                process_jsx(jsx_registry_path)
+            except ImportError:
+                pass
             collect_app_asset_src(app_asset_dirs, 'js')
 
     ### BUILD
 
     if DO_BUILD:
+        
+        # Build a bundle from the LESS, if present, then append the Stylus bundle, if present.
+        os.unlink(os.path.join(css_dir, 'bundle.css'))
```

Comment:
If css_dir is still None, what will this do?

Reply:
If we haven't found any less or stylus files, should we still delete an existing bundle.css file?

Reply:
That probably isn't going to be the case anywhere we use this, but its not unreasonable to account for as a safety check. Will do!